lib/src/child-writer/index.js

```diff
@@ -56,20 +58,46 @@ const EXIT_CODES = require('../exit-codes');
 exports.write = (image, drive, options) => {
   const emitter = new EventEmitter();
 
-  utils.getTemporaryLogFilePath().then((logFile) => {
-    const argv = utils.getCLIWriterArguments({
-      entryPoint: rendererUtils.getApplicationEntryPoint(),
-      logFile: logFile,
-      image: image,
-      device: drive.device,
-      validateWriteOnSuccess: options.validateWriteOnSuccess,
-      unmountOnSuccess: options.unmountOnSuccess
-    });
+  const argv = utils.getCLIWriterArguments({
+    entryPoint: rendererUtils.getApplicationEntryPoint(),
+    image: image,
+    device: drive.device,
+    validateWriteOnSuccess: options.validateWriteOnSuccess,
+    unmountOnSuccess: options.unmountOnSuccess
+  });
+
+  // There might be multiple Etcher instances running at
+  // the same time, therefore we must ensure each IPC
+  // server/client has a randomly generated name.
```

Comment:
If there is a malicious program running on the machine, I'm sure it has better things to do than mess with Etcher :laughing: Maybe include the string "Etcher" to prevent other processes which also happen to use numbers from interfering.